packages/web-app-files/src/store/actions.js

```diff
@@ -484,12 +484,13 @@ export default {
     commit('CLEAR_RESOURCES_TO_DELETE_LIST')
   },
 
-  async loadIndicators({ dispatch, commit }, { client, currentFolder }) {
+  loadIndicators({ dispatch, commit }, { client, currentFolder }) {
     // kind of bruteforce for now: remove the shares for the current folder and children, reload shares tree for the current folder.
     // TODO: when we refactor the shares tree we want to modify shares tree nodes incrementally during adding and removing shares, not loading everything new from the backend.
     commit('SHARESTREE_PRUNE_OUTSIDE_PATH', dirname(currentFolder))
-    await dispatch('loadSharesTree', { client, path: currentFolder })
-    commit('LOAD_INDICATORS')
+    const promiseLoad = dispatch('loadSharesTree', { client, path: currentFolder })
```

Comment:
the reason why this was blocking is that we do not want to render the current view twice or more often, the idea here is to use more performant propfind solution to get needed share indicator information. We will do that on our end.

Reply:
Ah, makes sense... Yes, I'm waiting for this propfind with the sharing info combined.